_pkgdown.yml

```diff
@@ -14,51 +19,12 @@ template:
       creator: "@rstudio"
       card: summary
 
-development:
-  version_tooltip: "Development version"
-
 home:
-  strip_header: false
   links:
   - text: Learn more
     # if relevant, set url to another website i.e https://rmarkdown.rstudio.com
     href: ""
 
-# structure of website themed with quillt
-navbar:
-  title: ~
-  type: default
-  structure:
-    left:  [intro, examples, articles]
-    right: [reference, news, github]
-  components:
-    home: ~
-    examples:
-      text: Examples
-      href: articles/articles/examples.html
-    articles:
-      text: Articles
-      menu:
-        - text: Testing quillt style
-          href: articles/articles/test-style.html
-    reference:
-      text: Reference
-      href: reference/index.html
-    github:
-      icon: fab fa-github fa-lg
-      href: https://github.com/rstudio/quillt
-    news:
-      text: News
-      menu:
-        - text: "Changelog"
-          href: news/index.html
-        - text: "------------------"
-        - text: "Blog posts"
-        # Add blog post where a released version is mentioned.
-        - text: "No blog post"
-        # - text: Version 0.2
-        #   href: <url>
```

Comment:
Here we need probably to keep some part of this. 

If we don't tweak the navbar template anymore, only two current menu are customized 
* Example: We wanted to have a quick way to access a showcase gallery of example of outputs formats. Without going explicitly into the Articles sections. Wrong idea ? 
* News sections: We didn't want the changelog only and wanted the blog post section, but differently than the `releases` feature included in pkgdown already - as we didn't have one post per release. 

Apart from that, the rest is not needed. Probably there to be explicit regarding what will be done by default (in case default was changed)

We can definitely reconsider all this or leave it out from the template and customize on a per project basis. It is not part of the theme directly.